global.json

```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "6.0.100",
+    "version": "8.0.0",
```

Comment:
@MangelMaxime Should be 8.0.100, no?

Reply:
You are right indeed, I suppose because we have `"rollForward": "latestMinor"` it didn't break on me.

But better to use a real version